src/StateManager.ts

```diff
@@ -718,7 +761,7 @@ export default class StateManager {
     }
 
     // https://stackoverflow.com/a/77837006
-    if ((ev.metaKey || ev.ctrlKey) && ev.key == "z") {
+    if ((ev.metaKey || ev.ctrlKey) && ev.key == "z" && !this.debugActive) {
```

Comment:
Replace with early-exit clause.
You may be able to listen for the left/right arrow key events in here, and pass them to the debugger.